### ISSUE TITLE
TypeError: Cannot read property 'insertAfter' of undefined

### ISSUE THREAD

**Issue by dave105010** [NMFR/optimize-css-assets-webpack-plugin#168]
Hi, I keep getting this error.
I did "yarn cache clean"
Reinstalled everything and still cannot figure out why this is failing.

```
Done in 38.91s.
Compiling...
Compilation failed:
warning package.json: No license field
TypeError: Cannot read property 'insertAfter' of undefined
    at node_modules/optimize-css-assets-webpack-plugin/node_modules/css-declaration-sorter/src/main.cjs:372:22
    at Array.forEach (<anonymous>)
    at processCss (node_modules/optimize-css-assets-webpack-plugin/node_modules/css-declaration-sorter/src/main.cjs:369:12)
    at node_modules/optimize-css-assets-webpack-plugin/node_modules/css-declaration-sorter/src/main.cjs:304:21
    at async LazyResult.runAsync (node_modules/optimize-css-assets-webpack-plugin/node_modules/postcss/lib/lazy-result.js:431:15)
    at async Promise.all (index 0)
```

**Comment by NMFR** [comment]
Should be fixed in 5.0.8, reopen otherwise please.

**Comment by dave105010** [comment]
> Should be fixed in 5.0.8, reopen otherwise please.

It's resolved. Thank you!!

**Comment by Siilwyn** [comment]
The underlying issue https://github.com/Siilwyn/css-declaration-sorter/issues/170 has now been fixed, looking at the changes the downgrade can be reverted. @NMFR
